### ISSUE TITLE
Ask "Gender" next to "Sex assigned at birth"

### ISSUE THREAD

**Issue by aliciabeckett-gov** [CDCgov/prime-simplereport#2113]

**Comment by aliciabeckett-gov** [comment]
From some SOGI work USDS did on a different sprint:

'This is the best version I've seen so far (copied from a researcher we worked with on our SOGI sprint):
What is your gender identity? (Choose all that apply). Add reasoning for why this question is being asked. Provide contact for feedback. Examples of terms that could be added in the United States with feedback from those communities when appropriate: Two-Spirit, Nadleeh, Muxe, Biza’ah, Winkte, Palao’ana, and Māhū.
Female, Woman, or Girl
Male, Man, or Boy
Nonbinary
Questioning
A gender identity not listed (please specify): ____________________________
Prefer not to disclose


What is your assigned gender at birth? (typically this is the gender marker which appears on your original birth certificate) (Choose one). Add reasoning for why this question is being asked. Provide contact for feedback.
Female (‘F’)
Male (‘M’)
X (‘X’)
Unsure
I was not assigned a gender at birth
A gender assigned at birth not listed (please specify): ____________________
Prefer not to disclose



So basically whatever you do, try to get as close to that as you can given your constraints

**Comment by aliciabeckett-gov** [comment]
We need to figure out how to send this to ReportStream as an additional field as well @nullflux

**Comment by kenieh** [comment]
I'm meeting with Jeremy on Friday to get more details. Also reached out to him if we can get in touch with one of the reearchers on the past discovery sprint.

**Comment by kenieh** [comment]
Should revisit as a team eventually to create a more inclusive + equitable experience for our users.

**Comment by mehansen** [comment]
This has been completed as part of the effort to support HIV
